_build_cfg.yml

```diff
@@ -449,10 +449,17 @@ Topics:
         File: decision_server
       - Name: JBoss Data Grid
         File: data_grid
+  - Name: OpenShift V3 Online
+    Dir: online_images
+    Distros: openshift-online
+    Topics:
+      - Name: Overview
+        File: index
```

Comment:
The developers_cli topic also needs to be added earlier up in the "Getting Started" section in this file.